sql-cli/tests/test___main__.py

```diff
@@ -114,6 +114,12 @@ def test_version():
     assert f"Astro SQL CLI {__version__}" in result.stdout
 
 
+def test_version_json_output():
+    result = runner.invoke(app, ["version", "--json"])
+    assert result.exit_code == 0
+    assert f'{{"version": "{__version__}"}}' in result.stdout
```

Comment:
```suggestion
    assert f'"version": "{__version__}"' in result.stdout
```
Does it make sense to check for a particular key/value pair instead of the whole json object? That way it is more clear what you expect it to contain. I don't know 🤷 On the other side the entire stdout should be a valid json.